lib/registrations/registration_checker.rb

```diff
@@ -116,7 +116,7 @@ def validate_update_status!(new_status, competition, current_user, target_user,
 
         if new_status == Registrations::Helper::STATUS_ACCEPTED && competition.competitor_limit_enabled?
           raise WcaExceptions::RegistrationError.new(:forbidden, Registrations::ErrorCodes::COMPETITOR_LIMIT_REACHED) if
-            competition.registrations.competing_status_accepted.count >= competition.competitor_limit
+            competition.registrations.competing_status_accepted.competing.count >= competition.competitor_limit
```

Comment:
why not use the method you just created above?

Reply:
Good point, changed!